worlds/stardew_valley/__init__.py

```diff
@@ -45,15 +45,25 @@ class StardewWebWorld(WebWorld):
     options_presets = sv_options_presets
     option_groups = sv_option_groups
 
-    tutorials = [
-        Tutorial(
-            "Multiworld Setup Guide",
-            "A guide to playing Stardew Valley with Archipelago.",
-            "English",
-            "setup_en.md",
-            "setup/en",
-            ["KaitoKid", "Jouramie", "Witchybun (Mod Support)", "Exempt-Medic (Proofreading)"]
-        )]
+    setup_en = Tutorial(
+        "Multiworld Setup Guide",
+        "A guide to playing Stardew Valley with Archipelago.",
+        "English",
+        "setup_en.md",
+        "setup/en",
+        ["KaitoKid", "Jouramie", "Witchybun (Mod Support)", "Exempt-Medic (Proofreading)"]
+    )
+
+    setup_fr = Tutorial(
+        "Guide de configuration MultiWorld",
+        "Un guide pour configurer Stardew Valley sur Archipelago",
```

Comment:
Ce qui est recommandé par le [guide de traduction](https://docs.google.com/document/d/1feGCth4_1e9kkyx_zEGNaAfl0TnvxP7oDTyNvkq2hSY/edit), c'est d'ajouter les autres langues comme des nouvelles lignes sous le guide existant. Ça se fait automatiquement lorsque le `tutorial_name` et `description` sont les mêmes.

```suggestion
        setup_en.tutorial_name,
        setup_en.description,
```